integrationTests/websocket/testNotifierWithWebsockets_test.go

```diff
@@ -608,7 +655,9 @@ func TestNotifierWithWebsockets_AllEvents(t *testing.T) {
 			HeaderType:  string(core.ShardHeaderV2),
 			HeaderHash:  blockHash,
 			Body: &block.Body{
-				MiniBlocks: make([]*block.MiniBlock, 1),
```

Comment:
Why was the expected len changed here and also from other tests?

Reply:
the expected len was not changed, only that we have now a non nil MiniBlock instance, it the previous implementation with http, block Body was not being used at all so it was not panicking until now